resources/lib/sutils.py

```diff
@@ -131,23 +131,27 @@ def scanRunning(self):
         return (xbmc.getCondVisibility('Library.IsScanningVideo') or
                 xbmc.getCondVisibility('Library.IsScanningMusic'))
 
-    def getBBDB(self, name):
-        name = util.request('http://csfd.bbaron.sk/find.php?' +
-                            urllib.urlencode({'sosac': 1, 'name': name}))
-        if name != '':
-            return self.getTVDB(name, 1)
-        return None
-
-    def getTVDB(self, name, level=0):
-        data = util.request('http://thetvdb.com/api/GetSeries.php?' +
-                            urllib.urlencode({'seriesname': name, 'language': 'cs'}))
-        try:
-            tvid = re.search('<id>(\d+)</id>', data).group(1)
-        except:
-            if level == 0:
-                tvid = self.getBBDB(name)
-            else:
+    def getTVDB(self, name, id):
+        if id:
+            data = (util.request('http://thetvdb.com/api/GetSeriesByRemoteID.php?imdbid=tt' +
+                    id + '&language=all'))
```

Comment:
Kvůli rozepsání na více řádků, kdy jsem se dočetl, že best practice je to dát celé do závorky jako jeden logický blok.
A je preferováno např. místo ukončení řádku lomitky.
Klidně je ale odstraním pokud beautifier nebude pak řvát.